gradio/interface.py

```diff
@@ -324,6 +328,8 @@ def clean_html(raw_html):
             if analytics_enabled is not None
             else os.getenv("GRADIO_ANALYTICS_ENABLED", "True") == "True"
         )
+        if not self.analytics_enabled:
```

Comment:
We can actually delete all of these lines:

```py
        self.analytics_enabled = (
            analytics_enabled
            if analytics_enabled is not None
            else os.getenv("GRADIO_ANALYTICS_ENABLED", "True") == "True"
        )
        if not self.analytics_enabled:
            os.environ["HF_HUB_DISABLE_TELEMETRY"] = "True"
```

Because `Interface` calls the `Blocks` constructor and passes in `analytics_enabled`